### PR TITLE
Basic translation functionality on user pages

### PR DESCRIPTION
**Pull request by osjuga**
- Translation works by URL or by hard-coded links (any language by URL, provided languages by link)
- TODO: persist by user, work across different pages, some other setting (rather than links)

### CODE REVIEW

**Review by caraujo777 — APPROVED** [review]
nice!!

File: src/main/java/com/google/codeu/servlets/MessageServlet.java [hunk 1]
```diff
@@ -111,4 +121,19 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
...
   }
+
+  private void translateMessages(List<Message> messages, String targetLanguageCode) {
+
+    Translate translate = TranslateOptions.getDefaultInstance().getService();
+
+    for (Message message : messages) {
+      String originalText = message.getText();
+
+      Translation translation =
```
Comment (gauravmishra) [inline src/main/java/com/google/codeu/servlets/MessageServlet.java:132]: nit: this formatting looks a little off...run the formatted on this file? ditto below.

File: src/main/webapp/js/user-page-loader.js [hunk 2]
```diff
@@ -110,9 +114,21 @@ function buildMessageDiv(message) {
   return messageDiv;
 }
 
+function buildLanguageLinks() {
+  const userPageUrl = '/user-page.html?user=' + parameterUsername;
+  const languagesListElement = document.getElementById('languages');
+  languagesListElement.appendChild(createListItem(createLink(userPageUrl + '&language=en', 'English')));
```
Comment (gauravmishra) [inline src/main/webapp/js/user-page-loader.js:120]: ditto

**Review by gauravmishra — APPROVED** [review]
Nice!

File: src/main/webapp/js/user-page-loader.js (hunk 2 above)
Comment (osjuga) [reply src/main/webapp/js/user-page-loader.js:120]: This is Javascript so I'm not sure if it's supposed to/will work as expected with the Java reformatter. What looks off about this chunk of code?

File: src/main/java/com/google/codeu/servlets/MessageServlet.java (hunk 1 above)
Comment (osjuga) [reply src/main/java/com/google/codeu/servlets/MessageServlet.java:132]: I ran the formatter and made arbitrary changes and it changed back to this. Is there anything that looks wrong in particular? I can check the style guide manually.

**Pull request merged**